code/modules/reagents/reagents/food-Drinks.dm

```diff
@@ -2957,15 +2960,36 @@
 	glass_unique_appearance = TRUE
 	glass_icon_state = "fanciulli"
 	glass_name = "Fanciulli"
-	glass_desc = "What if the Manhattan cocktail <b>actually</b> used a digestiff? Helps you sober up, but makes you sluggish."
+	glass_desc = "What if the Manhattan cocktail <b>actually</b> used a digestif? Helps you sober up, but makes you sluggish."
 
 /datum/reagent/drink/fanciulli/affect_ingest(var/mob/living/carbon/M, var/alien, var/effect_multiplier)
+	..()
 	M.nutrition = max(M.nutrition - 1 * effect_multiplier, 0)
 	M.dizziness = 0
 	M.stuttering = 0
 	M.confused = 0
 	M.Weaken(2)
 
+/datum/reagent/ethanol/jagerbomb
```

Comment:
This is to op

Reply:
It's also strong as fuck, and limited to a very niche energy drink you can't mass produce though